app/src/main/java/com/mxt/anitrend/model/entity/anilist/meta/TitleBase.java

```diff
@@ -70,15 +70,18 @@ public String getOriginal() {
     }
 
     public String getAlternativeFormatted() {
-        StringBuilder builder = new StringBuilder();
-        if (alternative != null)
```

Comment:
I think null check here was only working around bug solved in #403, should I remove it?

Reply:
I think you can leave it in, after all the `alternative` is nullable according to the graphql schema

Reply:
Actually, I guess we could just use string join instead of builder too.

Reply:
Probably for the best to avoid the overhead of creating a builder

Reply:
Ok, now it's cool.